src/MCPClient/lib/settings/common.py

```diff
@@ -16,20 +16,20 @@
 # along with Archivematica.  If not, see <http://www.gnu.org/licenses/>.
 
 import os, sys, ConfigParser
-import django_mysqlpool
 
 # Get DB settings from main configuration file
 config = ConfigParser.SafeConfigParser()
 config.read('/etc/archivematica/archivematicaCommon/dbsettings')
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django_mysqlpool.backends.mysqlpool',
+        'ENGINE': 'django.db.backends.mysql',
         'NAME': config.get('client', 'database'),          # Or path to database file if using sqlite3.
         'USER': config.get('client', 'user'),              # Not used with sqlite3.
         'PASSWORD': config.get('client', 'password'),      # Not used with sqlite3.
         'HOST': config.get('client', 'host'),              # Set to empty string for localhost. Not used with sqlite3.
         'PORT': '',                                        # Set to empty string for default. Not used with sqlite3.
+        'CONN_MAX_AGE': 500,
```

Comment:
My initial assumption was that setting an actual limit like 500 seconds prevents a runaway query from chewing up one connection for ever.  If there is a query that takes longer than 500 seconds, it will time out and die, causing an exception, with this setting.  

I don't know enough about how Django handles edge cases like connections that die due to bugs in the driver, or network issues etc.  Probably it works just fine.

I tested with a value of 500, it would be good to test with other values. I suggest we are just looking for a default value that is known to work and the user should/could change as needed.

Perhaps I should change 500 to 'config.get('client', 'conn_max_age') and set it in the config file?